### PR TITLE
fix(plugin-meetings): use new webrtc-core effects APIs

### PR DESCRIPTION
**Pull request by brycetham**
# COMPLETES [WEBEX-365024](https://jira-eng-gpk2.cisco.com/jira/browse/WEBEX-365024)

## This pull request addresses

Updating the SDK to use the new webrtc-core effects APIs described here: https://github.com/webex/webrtc-core/pull/68.

## by making the following changes

Updating the sample app and `webrtc-core.ts`.

### Change Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Tooling change
- [ ] Internal code refactor

## The following scenarios where tested

1. Manually tested on the WCME level.
2. Manually tested on the sample app by switching between all audio and video effects.

### I certified that

- [x] I have read and followed [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request)
- [x] I discussed changes with code owners prior to submitting this pull request

- [x] I have not skipped any automated checks
- [x] All existing and new tests passed
- [x] I have updated the documentation accordingly

---

Make sure to have followed the [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request) before submitting.

### CODE REVIEW

**Comment by brycetham** [comment]
Keeping this as draft until https://sqbu-github.cisco.com/CPaaS/web-client-media-engine/pull/309 and the associated internal-media-core PR has been merged.

**Comment by brycetham** [comment]
I've added a polyfill for crypto since the new effects lib changes use the latest uuid version, which requires crypto.

File: packages/@webex/plugin-meetings/package.json
Comment (chrisadubois) [inline packages/@webex/plugin-meetings/package.json:35]: q: why was this added?

**Review by chrisadubois — COMMENTED** [review]
I don't think the webex.min.js and webex.min.js.map are intended to be pushed

File: docs/samples/browser-plugin-meetings/app.js
Comment (chrisadubois) [inline docs/samples/browser-plugin-meetings/app.js:266]: is the crypto thing for this change? how do we properly access the url now? maybe a comment here would help? is it because the effect lib now does this internally?

**Review by chrisadubois — COMMENTED** [review]
Code looks good, just want to understand questions then can approve.

File: packages/@webex/plugin-meetings/package.json
Comment (chrisadubois) [reply packages/@webex/plugin-meetings/package.json:35]: nvm i see your comment above

**Review by chrisadubois — APPROVED** [review]
Looks fine to me though it would be prudent to get an approval from @martroos or someone who worked on the effects closely. This is just an API change and I did test it locally that it was working.

File: docs/samples/browser-plugin-meetings/app.js
Comment (brycetham) [reply docs/samples/browser-plugin-meetings/app.js:266]: No, the crypto thing is for the unit tests (sorry, I should have clarified this in my comment above, I'll edit it). The URL change is because the original URLs don't actually exist, so I've changed them to use relative URLs instead. Let me know if this is a problem; they seem to work when running the samples locally.

**Comment by brycetham** [comment]
> I don't think the webex.min.js and webex.min.js.map are intended to be pushed

Gah, I never remember whether or not they need to be pushed. I can revert them.

**Review by marcin-bazyl — APPROVED** [review]
I've only had a very quick look and didn't test with corresponding web app changes

**Review by martroos — APPROVED** [review]
The changes look fine, but this does require changes in web client so that it is able to upgrade the SDK (due to the breaking change in API). In order to test, and in preparation for this to go in, I have created a relevant PR for web client, so once this is merged we can make use of it. After making the changes, the web client appears to work well.

**Comment by chrisadubois** [comment]
@martroos I believe bryce had already had a web client PR for the above: https://sqbu-github.cisco.com/WebExSquared/webex-web-client/pull/5239

**Review by arun3528 — APPROVED** [review]

**Pull request merged**
